### PR TITLE
Improve sign/verify message ui

### PR DESCRIPTION
**Pull request by reyraa**
Coses #150

### CODE REVIEW

File: src/components/signVerify/signMessage.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import lisk from 'lisk-js';
+import './sign-message.less';
```
Comment (slaweet) [inline src/components/signVerify/signMessage.js]: Just for the sake of readability I would do this import also in `src/components/signVerify/verifyMessage.js`
And also rename `sign-message.less` to `signVerifyMessage.less`

Comment (reyraa) [inline src/components/signVerify/signMessage.js]: Thought it's redundant, but I've added.

**Comment by reyraa** [comment]
I don't like the way action bar looks in signMessage, but currently nothing comes into my mind. @slaweet  let me know if you have any suggestions.

**Comment by slaweet** [comment]
@reyraa I tried to improve how the action bar looks, in a commit to this branch.

**Pull request merged**